ledger/catchpointwriter_test.go

```diff
@@ -653,101 +684,186 @@ func TestExactAccountChunk(t *testing.T) {
 	cph := testWriteCatchpoint(t, dl.validator.trackerDB().Rdb, catchpointDataFilePath, catchpointFilePath, 0)
 	require.EqualValues(t, cph.TotalChunks, 1)
 
-	l := testNewLedgerFromCatchpoint(t, catchpointFilePath)
+	l := testNewLedgerFromCatchpoint(t, catchpointFilePath, dl.generator.trackerDB().Rdb)
 	defer l.Close()
 }
 
 func TestCatchpointAfterTxns(t *testing.T) {
 	partitiontest.PartitionTest(t)
 	t.Parallel()
 
-	genBalances, addrs, _ := ledgertesting.NewTestGenesis()
-	dl := NewDoubleLedger(t, genBalances, protocol.ConsensusFuture)
-	defer dl.Close()
+	// Exercises interactions between transaction evaluation and catchpoint
+	// generation to confirm catchpoints include expected transactions.
+	t.Run("chunks", func(t *testing.T) {
```

Comment:
Are we gaining something from pushing things into a `t.Run`? It obscures the diff, and I don't think it has the isolation properties you might be hoping for anyway.

Reply:
@jannotti It's subjective - I think there are 2 discrete test cases here.  I felt they can be left under the same test name (`TestCatchpointAfterTxns`) and separated by `t.Run`.

The PR's initial commit shows everything inline as 1 test case without `t.Run`.  As mentioned in the PR description, I'm willing to remove `t.Run` if reviewers feel strongly.

Reply:
@jannotti Per our verbal, https://github.com/algorand/go-algorand/pull/4818/commits/c9425ed9d67ae0e8e4d301f7b4e844254914a6b9 removes `t.Run` in favor of discrete tests.